opus/application/test_api/api_test_helper.py

```diff
@@ -1,12 +1,15 @@
 # opus/application/test_api/api_test_helper.py
+import base64
 import difflib
 from io import BytesIO
 import json
 import os
 import re
 import tarfile
 import zipfile
+from PIL import Image
 
+import numpy as np
```

Comment:
numpy is not in requirements.txt. It is brought in second-hand from other modules. It needs to be in the primary requirements.